.github/workflows/test-integration.yml

```diff
@@ -0,0 +1,34 @@
+name: test-integration
+on:
+  pull_request:
+  push:
+    branches: [ master, develop ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ runner.os }}-composer-
+      - name: Init environment
+        run: |
+          cd ci/docker
+          docker-compose up -d
+      - name: Install dependencies
+        run: |
+          cd ci/docker && docker-compose exec -T stepup-middleware bash -lc '
+          composer install
+          '
+      - name: Run test scripts
+        run: cd ci/docker && docker-compose exec -T stepup-middleware bash -lc ' composer test '
+      - name: Output log files on failure
+        if: failure()
+        run: cd ci/docker && docker-compose exec -T stepup-middleware -c 'cat var/log/webtest.log'
```

Comment:
This will not work here. You are explicitly running tests in the `test` env. Please update, and test if the log file shows useful information when an erroneous situation occurs.

Reply:
has been changed to the correct var/log/dev.log file